src/main/java/com/openshift/internal/restclient/model/ReplicationController.java

```diff
@@ -364,4 +366,24 @@ public void addVolume(IVolumeSource volumeSource) {
 		}
 		volList.add(ModelNode.fromJSONString(volumeSource.toJSONString()));
 	}
+
+    @Override
+    public void setServiceAccount(String serviceAccount) {
```

Comment:
Thanks for the info, I was confused because the openshift docs show both "serviceAccount" and "serviceAccontName" should be fixed now with get/set test.